docs/AboutUs.adoc

```diff
@@ -11,46 +11,40 @@ We are a team based in the http://www.comp.nus.edu.sg[School of Computing, Natio
 
 == Project Team
 
-=== John Doe
-image::damithc.jpg[width="150", align="left"]
```

Comment:
We might want to remove the profile pictures of the se-edu team as they're no longer needed any more I believe.

Reply:
Updated in 729a67d